sensor_msgs/PointCloud2 map
---
geometry_msgs/PoseArray grasp_candidates
